Lib/platform.py

```diff
@@ -334,6 +334,30 @@ def _syscmd_ver(system='', release='', version='',
     (6, None): "post2012ServerR2",
 }
 
+def win32_is_iot(isIot=False):
+    if win32_editionId() in ('IoTUAP', 'NanoServer', 'WindowsCoreHeadless', 'IoTEdgeOS'):
+        return True
+    return False
+
+def win32_editionId(editionId=''):
```

Comment:
Can we drop `Id` here and just have `win32_edition`?

Reply:
sounds good